.github/workflows/terraform-apply-env.yml

```diff
@@ -50,13 +21,14 @@ jobs:
       TERRAFORM_PRE_RUN: |
           apt-get update
           apt-get install -y zip python
-      TF_VAR_postgrest_image: ${{ needs.fetch-versions.outputs.postgrest }}
-      TF_VAR_clamav_image: ${{ needs.fetch-versions.outputs.clamav }}
+      TF_VAR_postgrest_image: 'ghcr.io/gsa-tts/fac/postgrest:latest'
```

Comment:
For [the reasons discussed in Slack](https://gsa-tts.slack.com/archives/C03QCP4FSN8/p1689279742600559?thread_ts=1689273606.817079&cid=C03QCP4FSN8), I actually think these belong directly in the code. That way if we ever need to pin to a particular version for some reason, we do that explicitly via PR and promote it through our environments just like any other change. (That could happen as a separate PR from this workflow refactor, though.)

Reply:
this item was mainly to remove outdated code and their respective versions. If we wish to use this, we can use a hardcoded `YYYYMMDD` instead of `latest`